data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/migrated/schema/HttpErrorResponse.json

```diff
@@ -1,9 +1,9 @@
 {
   "title" : "HttpErrorResponse",
   "type" : "object",
-  "required" : [ "status", "message" ],
+  "required" : [ "code", "message" ],
   "properties" : {
-    "status" : {
+    "code" : {
```

Comment:
beacon-rest-api requires that this is code, we had a ticket about it, but it looks like the new httpErrorResponse still had status... haven't renamed the object getter but could @ajsutton  - thoughts?

Reply:
Makes sense to be consistent and use code everywhere I'd say.